core/modules/user/user.tokens.inc

```diff
@@ -73,15 +73,23 @@ function user_token_info() {
     'description' => t('The URL of the account edit page.'),
     'type' => 'url',
   );
+
+  // This token is replaced only when the system sends an Account Cancellation
+  // email to a user.
+  // @see user_mail_tokens()
   $user['cancel-url'] = array(
     'name' => t('Account cancellation URL'),
-    'description' => t('The URL of the confirm delete page for the user account.'),
+    'description' => t('The URL of the confirm delete page for the user account. This token is only available when the system sends an Account Cancellation email to a user.'),
```

Comment:
We should not be using the word "User". Let's shorten this to "This token is only available when the system sends an Account Cancellation email."

Reply:
I've filed  a follow-up PR: https://github.com/backdrop/backdrop-issues/issues/5459